build-n-publish.sh

```diff
@@ -167,35 +172,44 @@ then
   canonicalTag=${versionFull}
 fi
 
-imageName=fromdoppler/doppler-for-shopify
```

Comment:
Now, we need to validate that image name param is sending (required). I can not find this validation on the script.

Reply:
now `imageName` is set by parameter in lines 34 to 36
the parameter is set in `Jenkinsfile` when call the script

Reply:
> Now, we need to validate that image name param is sending (required). I can not find this validation on the script.

You are right! good catch!

Reply:
[Done](https://github.com/MakingSense/doppler-for-shopify/pull/341/files#diff-eea8d343339fa863fa5643082802cfa59bbcdadc59e3aa6a37c707d9711d3a69R56-R62)